src/daos/build/install-single-host-dgw.sh

```diff
@@ -45,8 +22,19 @@ function usage()
     echo ""
 }
 
-BRANCH='add-daos-rgw-sal'
+function ceph_get()
+{
+    # wget --output-document=folder_free_space.sh https://github.com/zalsader/ceph/blob/docker-build/src/daos/folder_free_space.sh?raw=true
```

Comment:
What's the commented code?

Reply:
an example of the built command line.  It took me a while to figure it out so I wanted to leave it there